components/dot_menu/index.ts

```diff
@@ -79,7 +79,7 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
     const userId = getCurrentUserId(state);
     const channel = getChannel(state, post.channel_id);
     const currentTeam = getCurrentTeam(state) || {};
-    const team = getTeam(state, post.channel_id);
+    const team = getTeam(state, channel.team_id);
```

Comment:
Do DMs and GMs need to fall back to the current team ID here?

Reply:
The variable team is only used in the next line. Therefore, if it is a DM, it will use the currentTeam name.

Reply:
Ah, I see. I also didn't realize that the old code was just incorrect since it was requesting a team using a channel ID